packages/falcon-front-kit/src/SortOrder/areSortOrderInputsEqual.ts

```diff
@@ -0,0 +1,9 @@
+import { SortOrderInput } from '@deity/falcon-data';
+
+/**
+ * Determines if two `SortOrderInput` are equal from a value perspective
+ * @param {SortOrderInput?} item1
+ * @param {SortOrderInput? }item2
```

Comment:
```suggestion
 * @param item2
```

This is minor, but I think TS allows to use the typings directly from the definitions (method signature) to avoid double-typing. I saw it in other places, but this typo (extra space) made me do this suggestion 😄

Reply:
done, also turned off corresponding eslint rule 
```
jsdoc/require-param-type': 'off'
```